runtime/near-vm-runner/src/tests/runtime_errors.rs

```diff
@@ -77,6 +77,30 @@ fn test_multiple_memories() {
         ]);
 }
 
+#[test]
+fn test_multiple_memories() {
+    test_builder()
+        //      | wasm header  | version 1     | mem sect   | 1st mem  | 2nd mem    |
+        //      |              |               | 2 elt 7 byt|          |            |
+        .wasm(b"\x00\x61\x73\x6d\x01\x00\x00\x00\x05\x07\x02\x01\x01\x02\x01\x03\x04")
```

Comment:
Should `wat` be preferred instead? It should support encoding multiple memories just fine and should end up much more readable too?

Reply:
Oh I expected it to fail encoding, but it does work! Much better, thanks :)